convene-web/app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,9 @@
+  <header class="header">
+    <div class="items-container">
+      <h1><%= current_workspace.client.name %></h1>
+      <div>
+        <h3><%= current_workspace.name %></h3>
+        <h3 class="font-hairline">video meeting spaces</h3>
+      </div>
+    </div>
```

Comment:
Not blocking, but my understanding of the header layout @colombene put together is that the left hand side is the current workspace and the right hand side is the Convene branding. This appears to invert that.